internal/web_server/web/handlers/api/admin_api.go

```diff
@@ -166,7 +176,7 @@ func (aah *AdminApiHandler) uploadQuizImage(c echo.Context) error {
 	imageAsURL, err := url.Parse(imageURL)
 	if err != nil {
 		log.Println(err)
-		return utils.Render(c, http.StatusBadRequest, components.ErrorText(errorImageElementID, "Kunne ikke laste opp bilde"))
+		return utils.Render(c, http.StatusBadRequest, components.ErrorText(errorImageElementID, "Kunne ikke laste opp fullføre bildeopplastingen"))
```

Comment:
`Kunne ikke laste opp fullføre bildeopplastingen` is a nonsensical sentence. Fix.